docs/FAQ.md

```diff
@@ -14,7 +14,15 @@ you can also disable the plugins via environment variables `SW_AGENT_DISABLE_PLU
 
 #### Q: How to build from sources?
 
-#### A: If you want to build the SkyWalking Python Agent from source codes, for some reasons, for example, you want to try the latest features that're not released yet, please clone the source codes from GitHub and `make install` it:
+#### A: Download the source tar from [the official website](http://skywalking.apache.org/downloads/), and run the following commands to build from source
```

Comment:
We need a link or copy in the compiling doc as you release voting mail targeting there.